PluginsAndFeatures/azure-toolkit-for-intellij/.idea/vcs.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GithubSharedProjectSettings">
```

Comment:
Do we need to commit this?

Reply:
Not sure, IntelliJ keeps adding it. I'll revert the change.